docs/markdown/android/icon.md

```diff
@@ -94,6 +86,48 @@ Used sparingly to draw attention to an icon that might otherwise be missed.
 </Group>
 </TwoCol>
 
+### Color
+Icon colors are semantic—they have a specific meaning and aren't arbitrary. There is no disabled color for icons, as that is handled by the button state that an icon is in.
```

Comment:
semantic —they

typo

Reply:
thank you! what's the typo? a space before the em dash? Per content standards, no spaces: 
<img width="967" alt="image" src="https://user-images.githubusercontent.com/96082362/220421325-d69ee1e5-d4ec-4bee-9e71-a4aa1a353261.png">
https://gestalt.pinterest.systems/foundations/content_standards/grammar#Punctuation